### PR TITLE
Biomass_core.Rmd -> module manual.

### PR DESCRIPTION
**Pull request by CeresBarros**
* Biomass_core.Rmd is now a "complete" module manual with details on functioning, differences from LANDIS-II BSE and a thorough explanation of  module inputs

*TO DO:
- perhaps a better explanation of module outputs will be necessary
- Make Table 2 as wide as Table 1

### CODE REVIEW

**Review by eliotmcintire — APPROVED** [review]
Looks great! Amazing job.

**Review by achubaty — APPROVED** [review]

File: Biomass_core.R [hunk 1]
```diff
@@ -873,8 +896,8 @@ Init <- function(sim, verbose = getOption("LandR.verbose", TRUE)) {
     maxBiomass <- maxValue(sim$biomassMap)
     if (maxBiomass < 1e3) {
       if (verbose > 0) {
-        message(crayon::green("  Because biomassMap values are all below 1000, assuming that these should be\n",
-                              "    converted to tonnes/ha by multiplying by 100"))
+        message(crayon::green("  Because biomassMap values are all below 1000, assuming that these are on ton/ha.\n",
```
Comment (ianmseddy) [inline Biomass_core.R:899]: can we change ton/ha to Mg/ha (or at least tonne/ha)?

File: Biomass_core.R [hunk 2]
```diff
@@ -157,47 +161,49 @@ defineModule(sim, list(
...
+                              "based on identical `ecoregionGroup`, `speciesCode`, `age` and `B` composition,",
+                              "even if the user supplies other initial groupings (e.g., via the `Biomass_borealDataPrep`",
+                              "module.")),
     expectsInput("rasterToMatch", "RasterLayer",
-                 desc = "a raster of the studyArea in the same resolution and projection as biomassMap"),
+                 desc = "a raster of the `studyArea` in the same resolution and projection as `biomassMap`"),
     expectsInput("species", "data.table",
                  desc = paste("a table that has species traits such as longevity, shade tolerance, etc.",
-                              "Default is partially based on Dominic Cir and Yan's project"),
+                              "Default is partially based on Dominic Cir and Yan Boulanger's project"),
```
Comment (ianmseddy) [inline Biomass_core.R:174]: I believe it's Dominic Cyr, not Cir

**Review by ianmseddy — CHANGES_REQUESTED** [review]

File: Biomass_core.R (hunk 1 above)
Comment (ianmseddy) [reply Biomass_core.R:899]: actually we use tonnes/ha below this, so if we use Mg we should update both instances.

Comment (CeresBarros) [reply Biomass_core.R:899]: tonnes/ha now

**Pull request merged**